doc/releases.md

```diff
@@ -14,30 +14,33 @@ CDI adheres to the [semantic version definitions](https://semver.org/) format of
 
 ### Releasing a New Version
 
- The version number is tracked in several files in CDI as well as through a git tag.  To reduce the chance of human error, a help script is used to change the version in all known locations.
+Release branches are used to isolate a stable version of CDI.  Git tags are used within these release branches to track incrementation of Minor and Patch versions.  When a Major incrementation occurs, a new stable branch should be created corresponding to the release.
```

Comment:
is "incrementation" a word? :)

Reply:
Wellllll .... 185K hits on google, none of which is the oxford dictionary :P